internal/bgp/native/native.go

```diff
@@ -98,9 +98,14 @@ func (sm *sessionManager) SyncExtraInfo(extras string) error {
 
 func (sm *sessionManager) SetEventCallback(func(interface{})) {}
 
+func (s *session) PeerLabelFor() string {
```

Comment:
yeah the "for" can be removed (in response to your comment 😅)

Reply:
done